buildbot/buildbot/worker.py

```diff
@@ -532,8 +479,7 @@ def stage_fpga_execute(db, config):
             )
 
 
-STAGES = (stage_unpack, stage_make, stage_seashell, stage_hls,
-          stage_fpga_execute)
+STAGES = (stage_unpack, stage_make, stage_hls, stage_fpga_execute)
```

Comment:
No need for parentheses if it all fits on one line.

Reply:
Woah, didn't know you define arrays like that!